src/riak_core_handoff_cli.erl

```diff
@@ -146,11 +146,11 @@ details_usage() ->
 handoff_config(_Args, Flags) when length(Flags) > 1 ->
     [clique_status:text("Can't specify both --all and --node flags")];
 handoff_config(_Args, []) ->
-    clique_config:show(config_vars());
+    clique_config:show(config_vars(), []);
 handoff_config(_Args, [{all, _}]) ->
-    clique_config:show(config_vars() ++ ["-a"]);
+    clique_config:show(config_vars(), [{all, undefined}]);
```

Comment:
I wonder if it would be a good idea to pass through the value for `[{all, Arg}]` in the function head to clique_config:show/2 in case the value is ever anything other than `undefined`